### PR TITLE
Add moderator statement model to budgeting

### PR DESCRIPTION
**Pull request by kleingeist**
Add a moderator statement model and make it editable with a MultiForm on
the moderation page.

As the whole budget detail page is not styled yet, this PR does not include any style information.

### CODE REVIEW

File: apps/budgeting/models.py [hunk 1]
```diff
@@ -22,3 +25,11 @@ class Proposal(idea_models.AbstractIdea):
 
     def get_absolute_url(self):
         return reverse('budgeting:proposal-detail', args=[str(self.slug)])
+
+
+class ModeratorStatement(UserGeneratedContentModel):
+    proposal = models.OneToOneField(Proposal,
+                                    primary_key=True,
+                                    related_name="moderator_statement")
```
Comment (2e2a) [inline apps/budgeting/models.py]: Preferred indenting:
```
proposal = models.OneToOneField(
    Proposal,
```

File: apps/budgeting/models.py [hunk 2]
```diff
@@ -22,3 +25,11 @@ class Proposal(idea_models.AbstractIdea):
 
     def get_absolute_url(self):
         return reverse('budgeting:proposal-detail', args=[str(self.slug)])
+
+
+class ModeratorStatement(UserGeneratedContentModel):
```
Comment (2e2a) [inline apps/budgeting/models.py]: I think this model should be in the `moderatorfeedback` app.

File: apps/budgeting/forms.py [hunk 3]
```diff
@@ -12,12 +13,37 @@ class Meta:
         fields = ['name', 'description', 'category', 'budget']
 
 
-class ProposalModerateForm(forms.ModelForm):
-
-    def __init__(self, *args, **kwargs):
-        kwargs.pop('module')
-        super(ProposalModerateForm, self).__init__(*args, **kwargs)
+class ModeratorFeedbackForm(forms.ModelForm):
```
Comment (2e2a) [inline apps/budgeting/forms.py]: Better move to `moderatorfeedback` app.

File: apps/budgeting/forms.py [hunk 4]
```diff
@@ -12,12 +13,37 @@ class Meta:
...
 
     class Meta:
         model = models.Proposal
         fields = ['moderator_feedback']
+
+
+class ModeratorStatementForm(forms.ModelForm):
```
Comment (2e2a) [inline apps/budgeting/forms.py]: Better move to `moderatorfeedback` app

**Comment by kleingeist** [comment]
Need to evaluate form / view centric logic for creation

File: apps/budgeting/forms.py (hunk 4 above)
Comment (kleingeist) [inline apps/budgeting/forms.py]: as discussed, the simple moderator statement code retains in the budgeting app

File: apps/budgeting/forms.py (hunk 3 above)
Comment (kleingeist) [inline apps/budgeting/forms.py]: as discussed, the simple moderator statement code retains in the budgeting app

File: apps/budgeting/models.py (hunk 2 above)
Comment (kleingeist) [inline apps/budgeting/models.py]: as discussed, the simple moderator statement code retains in the budgeting app

**Comment by kleingeist** [comment]
i moved the statement creation logic from the view to the form to be more in line with the other code

**Comment by 2e2a** [comment]
lgtm

**Pull request merged**